_docs_integrate/attribute-introduction.md

```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
+This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
+This is because the LocalAttribute which is associated with the RelationshipAttribute and which is used as the source for sharing can only be available locally to the RelationshipAttribute's sharer.
 
-As already mentioned, the creation of a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) corresponds to the creation of one LocalAttribute for its `owner` and one [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) for the peer with whom the `owner` has established a Relationship. The peer shared RelationshipAttribute is the peer's LocalAttribute and therefore forms the counterpart of the own shared RelationshipAttribute. The `id` of a peer shared RelationshipAttribute is always the same as the `id` of the associated own shared RelationshipAttribute. As with the own shared RelationshipAttribute, the `shareInfo` property of a peer shared RelationshipAttribute is of course set. Within the `shareInfo.peer` property, the Address of the `owner` of the RelationshipAttribute is specified. In any case, the `shareInfo.sourceAttribute` property of a peer shared RelationshipAttribute is undefined. This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
+#### Own shared and peer shared ThirdPartyRelationshipAttributes
 
-Sometimes a peer shared RelationshipAttribute is referred to as a peer shared ThirdPartyRelationshipAttribute. This is the case if it can be interpreted as a counterpart of an own shared ThirdPartyRelationshipAttribute. Note that the `shareInfo.sourceAttribute` property of a peer shared ThirdPartyRelationshipAttribute is always undefined, as the own shared RelationshipAttribute used as the source can only be available locally to its owner.
-{: .notice--info}
+Note that it is possible to share the underlying RelationshipAttribute of an [own shared RelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#own-shared-and-peer-shared-relationshipattributes) with peers who are not involved in the Relationship in which the RelationshipAttribute exists, provided that the `confidentiality` of the RelationshipAttribute is not `"private"`.
+The sharing of a RelationshipAttribute by its `owner` with such a peer in turn leads to the creation of another own shared RelationshipAttribute in the wallet of the `owner`, which contains the Address of this peer within its `shareInfo.peer` property.
+Such an own shared RelationshipAttribute is referred to as an own shared ThirdPartyRelationshipAttribute.
```

Comment:
As you already introduced all the terms above, I'd directly use the correct one to avoid confusion. Namely:
"The sharing of a RelationshipAttribute by its owner with such a **third party** in turn leads to the creation of **an own shared ThirdPartyRelationshipAttribute** in the wallet of the owner,"

Reply:
I know what you mean and will take that into consideration. I think it depends on whether we want to treat "own shared ThirdPartyRelationshipAttributes" as a special case of "own shared RelationshipAttributes" or as a separate LocalAttribute class. What is your first opinion on this? We can probably only decide this once we have clarified the functionalities relating to ThirdPartyRelationshipAttributes in more detail.

Reply:
- 👍 
- Since you already divided the sections own/peer shared RelationshipAttributes and own/peer shared ThirdPartyRelationshipAttributes, I'd use these terms consistently strictly divided here.